components/SideNavigation.vue

```diff
@@ -201,9 +201,9 @@ export default Vue.extend({
         },
         {
           svg: 'CovidIcon',
-          title: this.$t('新型コロナウイルスの感染が判明した方へ'),
+          title: this.$t(' 新型コロナウイルス感染症で入院・宿泊療養される方へ'),
```

Comment:
```suggestion
          title: this.$t('新型コロナウイルス感染症で入院・宿泊療養される方へ'),
```

わあ、ごめんなさい！今度は余計なスペースが入っていたみたいです 🙇

Reply:
何度もすみません。修正しました！